### PR TITLE
[DOCS-7432] Add Healthcheck configuration

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by damianujma — APPROVED** [review]

**Review by mjstanford — APPROVED** [review]

File: search-enterprise/latest/admin/index.md [hunk 1]
```diff
@@ -496,3 +496,89 @@ The main use case to re-index only content or path is a fully metadata indexed r
...
+
+### Run Healthcheck Job
+
+Use the JConsole to run the Healthcheck.
+
+  **Note:** The MBean `Alfresco:Name=ElasticsearchHealthcheck` is exposed and allows you to manage the Healthcheck.
+
+1. Open a command prompt and `cd` to your JDK installation directory.
+
+2. Open the Java Monitoring & management Console window by entering: `jconsole`.
```
Comment (anxumalo) [inline search-enterprise/latest/admin/index.md:534]: I'm not sure what the correct capitalization is, but can we use initial capitals?
```suggestion
2. Open the Java Monitoring & Management Console window by entering: `jconsole`.
```

File: search-enterprise/latest/admin/index.md [hunk 2]
```diff
@@ -496,3 +496,89 @@ The main use case to re-index only content or path is a fully metadata indexed r
...
+
+1. Open the Admin Console, for more see [Launch Admin Console]({% link content-services/latest/admin/admin-console.md %}#launch-admin-console).
+
+2. In the **Repository Services** section, click **Search Service**.
+
+3. Select the **Service Status** tab.
+
+![health]({% link search-enterprise/images/health-check.png %})
+
+The Service Status tab gives information on the status of the latest Healthcheck execution, latest health check settings, and scheduled health check settings. At the bottom of the page there is a list of the latest Healthcheck events sorted by date. You can see ranges of nodes where discrepancies in the indexing were found. This means the range starts immediately after the previous correctly indexed node and finishes immediately before the first next correctly indexed node. You can also see the number of issues found within each range.
```
Comment (anxumalo) [inline search-enterprise/latest/admin/index.md:563]:
```suggestion
The **Service Status** tab gives information on the status of the latest Healthcheck execution, latest health check settings, and scheduled health check settings. At the bottom of the page there is a list of the latest Healthcheck events sorted by date. You can see ranges of nodes where discrepancies in the indexing were found. This means the range starts immediately after the previous correctly indexed node and finishes immediately before the first next correctly indexed node. You can also see the number of issues found within each range.
```

**Review by anxumalo — CHANGES_REQUESTED** [review]
Cool feature - nice read! Are we writing "healthcheck" as one word or two separate words? The majority of times you've written it as one word. Let's keep it consistent

**Comment by oliveralfresco2018** [comment]
> Cool feature - nice read! Are we writing "healthcheck" as one word or two separate words? The majority of times you've written it as one word. Let's keep it consistent

No worries, done.

**Pull request merged**